packages/expo/src/AR.ts

```diff
@@ -672,3 +672,15 @@ export function isFrontCameraAvailable(): boolean {
 export function isRearCameraAvailable(): boolean {
   return isConfigurationAvailable(TrackingConfiguration.World);
 }
+
+/* Legacy constants */
```

Comment:
Add today's date so that "legacy" is put into context and so we can remove these a year from now.